editor/package.json

```diff
@@ -41,28 +41,6 @@
     "fetch-tailwind-classes": "npx -s sh ts-node --pretty --require ignore-styles --transpile-only ./src/scripts/fetch-and-parse-tailwind-classes.ts && npx prettier --write ./src/core/third-party/tailwind-defaults.ts",
     "lint-editor": "lint-staged"
   },
-  "jest": {
```

Comment:
this was not used since we moved to jest.config.js, I removed it to avoid confusing people